### PR TITLE
fixed issues per Brand'ts comments

### PR DESCRIPTION
**Pull request by mmcxii**

### CODE REVIEW

File: client/src/Hooks/useLSUserInfo.ts [hunk 1]
```diff
@@ -8,10 +8,15 @@ export const useLSUserInfo = () => {
...
+    try {
+      if (lsLoginToken && lsUserInfo) {
+        const parsedData = JSON.parse(lsUserInfo);
 
-      dispatch({ type: 'LOG_USER_IN', payload: parsedData });
+        dispatch({ type: 'LOG_USER_IN', payload: parsedData });
+      }
+    } catch (error) {
+      localStorage.removeItem('loginToken');
+      localStorage.removeItem('userInfo');
```
Comment (mmcxii) [inline client/src/Hooks/useLSUserInfo.ts]: Does this look good? I've never used a trycatch block outside of an async function so I'm not sure if I need to do that too or if this is kosher.

File: client/src/Pages/Login/Login.tsx [hunk 2]
```diff
@@ -48,8 +50,10 @@ const Login: React.FC<Props> = () => {
       }
 
       localStorage.setItem('loginToken', loginToken);
-      localStorage.setItem('userInfo', JSON.stringify({ ...userData }));
+      localStorage.setItem('userInfo', JSON.stringify(userData));
```
Comment (mmcxii) [inline client/src/Pages/Login/Login.tsx]: For whatever reason TypeScript was complaining about the userData object having the type UserProps, however it seems to be okay with it now. I'm guessing that was just a TS thing, not an actual issue so I probably should have just restarted TypeScript to see if that fixed it. Regardless it seems to be working now.

File: client/src/Pages/Login/Login.tsx [hunk 3]
```diff
@@ -48,8 +50,10 @@ const Login: React.FC<Props> = () => {
...
       dispatch({ type: 'LOG_USER_IN', payload: userData });
+
+      return history.goBack();
```
Comment (mmcxii) [inline client/src/Pages/Login/Login.tsx]: Added this too, it just takes you back to wherever you were after logging in.

File: client/src/Hooks/useLSUserInfo.ts (hunk 1 above)
Comment (brandtkstrom) [reply client/src/Hooks/useLSUserInfo.ts]: Yea this looks good. Mainly needs to be around JSON.parse in case it throws an exception.

**Pull request merged**